liams-news/src/Components/ Comments/Comments.jsx

```diff
@@ -0,0 +1,54 @@
+import { useContext, useEffect, useState } from 'react'
+import UserContext from '../UserContext'
+import axios from 'axios'
+import { Link } from 'react-router-dom';
+import { NewCommentSection } from './Components/Comments'
+
+
+function Comments({ article_id }) {
+    const { user } = useContext(UserContext)
+
+    const [commentsList, setCommentsList] = useState([])
+    const [loading, setLoading] = useState(false)
+    const [updating, setUpdating] = useState(false)
+
+    useEffect(() => {
+        setLoading(true)
+        setUpdating(false)
+        let searchURL = `https://nc-news-liam.onrender.com/api/articles/${article_id}/comments`
```

Comment:
If this is never reassigned, it could be a `const` - you could consider extracting this into another file anyway